src/beautifiers/clang-format.coffee

```diff
@@ -0,0 +1,60 @@
+###
+Requires clang-format (https://clang.llvm.org)
+###
+
+"use strict"
+Beautifier = require('./beautifier')
+path = require('path')
+fs = require('fs')
+
+module.exports = class ClangFormat extends Beautifier
+
+  name: "clang-format"
+
+  options: {
+    "C++": true
+    "C": true
+    "Objective-C": true
```

Comment:
Thanks for pointing it out. Since clang-format uses its own configuration file, it does not make too much sense to accept options. And projects using clang-format usually provides their own configuration file. I will change it in a new commit.